Readme.md

```diff
@@ -114,6 +114,14 @@ The root node returned by `css.parse`.
 - stylesheet: `Object`:
   - rules: `Array` of nodes with the types `rule`, `comment` and any of the
     at-rule types.
+  - parsingErrors: `Array` of `Object`s. Errors collected during parsing when
+    option `silent` is true.
+    - `message`: `String`. The full error message with the source position.
+    - `reason`: `String`. The error message without position.
+    - `filename`: `String`.
+    - `line`: `Integer`.
+    - `column`: `Integer`.
+    - `source`: `String`. The portion of code.
```

Comment:
This belongs better in the “Errors” section above. Just keep line 117 and 118 here.